Add PostgreSQL by VSHN Backup Restore documentation (#34)
appuio/appcat-docs: docs/modules/ROOT/pages/vshn-managed/postgresql/backup.adoc

= Get Backup Details

> . Backup are enabled by default and your PostgreSQL instance should have the following backup configuration.

Reviewer: we should phrase it again as your version is not very good as well
```suggestion
. Backup are enabled by default and your PostgreSQL instance has the following backup configuration.
```

+
[source,yaml]
----
apiVersion: vshn.appcat.vshn.io/v1
kind: VSHNPostgreSQL
metadata:
  name: pgsql-app1-prod
  namespace: prod-app
spec:
  parameters:
    ...
    backup:
      schedule: 0 22 * * * <1>
    ...
----
<1> The time when the backup should be started.

. List your backups.
+
[source,bash]
----

> kubectl -n postgres get vshnpostgresbackups

Reviewer: it's an example namespace.

BACKUP NAME                                 DATABASE INSTANCE   STORED TIME               STATUS     AGE
pgsql-app1-prod-pk8k4-2023-03-05-13-05-00   pgsql-app1-prod     2023-03-05T13:05:15.755Z  Completed  38s
pgsql-app1-dev-pk8k4-2023-03-01-16-52-11    pgsql-app1-dev                                Failed     28s
----

. Choose one backup and get its details.
+
[source,bash]
----
kubectl -n postgres get vshnpostgresbackups pgsql-app1-prod-pk8k4-2023-03-05-13-05-00 -oyaml
----
+
[source,yaml]
----
apiVersion: api.appcat.vshn.io/v1
kind: VSHNPostgresBackup
metadata:
  name: pgsql-app1-prod-pk8k4-2023-03-05-13-05-00 <1>
  namespace: postgres
status:
  databaseInstance: pgsql-app1-prod <2>
  process:
    failure: "" <3>
    jobPod: pgsql-app1-prod-pk8k4-backup-27967024-6xjpq
    status: Completed
    timing:
      end: "2023-03-05T13:05:15.61228Z"
      start: "2023-03-05T13:05:09.519369Z"
      stored: "2023-03-05T13:05:15.755Z" <4>
----
<1> The name of the backup.
<2> The name of the database instance of this backup.

> <3> The failure message in case the backup job failed.

Reviewer 2: It failed in the past, not? So using the past tense makes more sense to me...

> <3> The failure message in case the backup job failed.

Reviewer: I feel the present tense works better, but my english is not great so for it's fine to use the past tense :)
